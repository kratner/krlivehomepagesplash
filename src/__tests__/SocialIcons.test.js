import React from "react";
import { shallow } from "enzyme";
import SocialIcons from "../components/SocialIcons";

describe("SocialIcons", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<SocialIcons iconName="twitter" />);
    expect(wrapper).toMatchSnapshot();
  });
});
