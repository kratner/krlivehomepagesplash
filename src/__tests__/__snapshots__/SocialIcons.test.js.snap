// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialIcons renders correctly 1`] = `ShallowWrapper {}`;
